### DISCUSSION TITLE
Add CRUD example

### DISCUSSION THREAD

**Discussion by lz1998** [tokio-rs/axum#674] [Ideas]
I have written `Book-Management` with 6 handlers: `register`, `login`, `create_book`, `search_book`, `update_book` and `delete_book`.

These handlers can help beginners learn how to write a simple CRUD website with `axum`, `sea-orm`, `jwt`.

Can I create a PR to add this example?

**Comment by davidpdrsn** [comment]
Depends on how big it is. Since examples also takes time to maintain I would like to keep them small. So it might be better to link to it from here https://github.com/tokio-rs/axum/blob/main/ECOSYSTEM.md
